README.md

```diff
@@ -1 +1,915 @@
+    - [The `model upload` command](#the-model-upload-command)
+  - [Examples of usage](#examples-of-usage)
+    - [Run Rasa X / Enterprise with a local Rasa Server](#run-rasa-x--enterprise-with-a-local-rasa-server)
+    - [Run Rasa X / Enterprise with mounted a local Rasa project](#run-rasa-x--enterprise-with-mounted-a-local-rasa-project)
+    - [Upgrade Rasa X / Enterprise version](#upgrade-rasa-x--enterprise-version)
+    - [Deploy Rasa X in one of the public cloud providers](#deploy-rasa-x-in-one-of-the-public-cloud-providers)
+    - [Upload a model to Rasa X](#upload-a-model-to-rasa-x)
+  - [Development](#development)
+    - [How to run it?](#how-to-run-it)
+    - [Kind cluster for developing purposes](#kind-cluster-for-developing-purposes)
+    - [Deploy Rasa X with mounted a local path](#deploy-rasa-x-with-mounted-a-local-path)
+    - [Open Rasa X in a web browser](#open-rasa-x-in-a-web-browser)
+    - [Deploy Rasa X with mounted a local path and a custom Docker image](#deploy-rasa-x-with-mounted-a-local-path-and-a-custom-docker-image)
+  - [License](#license)
+
+## Prerequisites
+
+- Kubernetes cluster >= 1.14+
+
+   or
+
+- kind (for local mode)
+
+## Installation
+
+### Linux / macOS
+
+- Binary downloads of `rasactl` can be found on [the Releases page](https://github.com/rasahq/rasactl/releases/latest).
+
+```text
+$ curl -L https://github.com/RasaHQ/rasactl/releases/download/0.0.7/rasactl_0.0.7_darwin_amd64.tar.gz
+$ tar -zxvf rasactl_0.0.7_darwin_amd64.tar.gz
+$ cp rasactl_0.0.7_darwin_amd64/rasactl /usr/local/bin/
+```
+
+- Using `brew`
+
+```text
+$ brew tap rasahq/rasactl
+$ brew install rasactl
+```
+
+## Before you start
+
+Below you can find several things that are good to know and keep in mind when you use `rasactl`.
+
+- `rasactl` executes an operation always only for a given deployment. The order in which `rasactl` determine a deployment to use:
+
+  1. A deployment name passed as an argument.
+  2. The `rasactl` checks if a `.rasactl` file exists in a current working directory, if yes, then a deployment defined in the file is used. The `.rasactl` file is created automatically when the `rasactl start --project` command is executed.
+  3. The `rasactl` checks if a default deployment is configured in the `rasactl.yaml` configuration file, if yes, then the default deployment is used. The default deployment can be set by using the `rasactl config use-deployment` command.
+  4. If there is only one deployment, then it's used.
+
+  You can use the [`rasactl list`](#the-list-command) command to check which deployment is used as the current one.
+
+  The `rasactl delete` command always requires explicitly pass a deployment name as an argument.
+
+- `rasactl` uses the [`rasa-x-helm`](https://github.com/RasaHQ/rasa-x-helm) chart to deploy Rasa X / Enterprise.
+- `rasactl` deploys Rasa X / Enterprise without Rasa Open Source Server. It's up to you to connect Rasa OSS with Rasa X / Enterprise deployment.
```

Comment:
```suggestion
- `rasactl` deploys Rasa X / Enterprise without a Rasa Open Source server. It's up to you to connect Rasa OSS with Rasa X / Enterprise deployment.
```
I would add links to the section here about "You can do this by <connecting to an existing one> or <configuring a local project> " etc